usr/share/rear/lib/global-functions.sh

```diff
@@ -821,37 +821,88 @@ function umount_davfs() {
 function umount_mountpoint() {
     local mountpoint="$1"
     local lazy="${2:-}"
+    local timeout_secs=2
 
+    contains_visible_char "$mountpoint" || BugError "umount_mountpoint() called with empty mountpoint argument '$mountpoint'"
+    test -d "$mountpoint" -o -b "$mountpoint" || Error "umount_mountpoint mountpoint '$mountpoint' neither directory nor block device"
+
+    # TODO: remove the test $lazy section after all mount functions and calls are fixed (or removed)
     if test $lazy ; then
```

Comment:
As far as I see the variable lazy is no longer used
in the current implementation of umount_mountpoint()